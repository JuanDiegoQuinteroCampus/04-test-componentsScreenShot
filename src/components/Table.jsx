import React from 'react'

export default function Table() {
  return (
    <>
    <h1>Holla</h1>
    <div>Table</div>
    </>
  )
}
