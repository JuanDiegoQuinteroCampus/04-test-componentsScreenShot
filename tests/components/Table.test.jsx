import { render } from "@testing-library/react"
import Table from "../../src/components/Table"

describe("Captura del componente <tabla.jsx>", ()=>{
    test("fotico :)", ()=>{
    let {container} = render(<Table/>);
        // expect(container).toMatchSnapshot();
        console.log(container);
        
    })
})