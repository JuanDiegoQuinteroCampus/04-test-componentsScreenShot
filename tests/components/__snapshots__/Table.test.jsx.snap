// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Captura del componente <tabla.jsx> fotico :) 1`] = `
<div>
  <div>
    Table
  </div>
</div>
`;
